testing/log4cxx/APKBUILD

```diff
@@ -0,0 +1,46 @@
+# Contributor: Russ Webber <russ@rw.id.au>
+# Maintainer: Russ Webber <russ@rw.id.au>
+pkgname=log4cxx
+pkgver=0.10.0
```

Comment:
A minor nit. I would quote pkgname and pkgver since those are strings.

Reply:
There are only 17 packages that quote versus 5089 that do not quote, that is just introducing useless fragmentation.

Reply:
I'm leaving this one.

Reply:
@russkel no problem

@maxice8 another way to reduce fragmentation is https://github.com/tcely/aports/tree/quote-pkgname-pkgver ;-)

Reply:
> https://github.com/tcely/aports/tree/quote-pkgname-pkgver ;-)

Very funny